frontend/js/helpers/Account.js

```diff
@@ -60,6 +60,10 @@ function updateUI(account,balance=0) {
     $("#addAcc").submit(async (event) => {
         event.preventDefault();
         const newName = $("#accountInput").val().trim();
+        if (accounts.some(account => account.username === newName)) {
```

Comment:
I found the problem in here.
Don't use accounts here instead get accounts data from api and compare to input value.